Updates/1559_Saltgurka-Molten_Core_Fixes.sql

```diff
@@ -0,0 +1,454 @@
+(@GUID,22,1080.754,-681.5721,-159.8726,100,1000),
+(@GUID,23,1080.708,-679.2873,-160.3567,100,1000),
+(@GUID,24,1082.191,-681.9566,-159.3777,100,9000),
+(@GUID,25,1079.69,-678.5046,-160.8126,100,8000),
+(@GUID,26,1080.177,-679.1766,-160.5332,100,1000),
+(@GUID,27,1078.341,-677.3896,-161.4327,100,9000),
+(@GUID,28,1079.791,-676.366,-161.2243,100,1000),
+(@GUID,29,1078.341,-677.3896,-161.4327,100,1000),
+(@GUID,30,1081.163,-683.5833,-159.3395,100,7000),
+(@GUID,31,1080.636,-682.1011,-159.7975,100,1000),
+(@GUID,32,1081.985,-678.5395,-160.1417,100,2000),
+(@GUID,33,1081.526,-679.8784,-159.9983,100,1000),
+(@GUID,34,1080.778,-681.3652,-159.9082,100,1000),
+(@GUID,35,1079.221,-677.4147,-161.1729,100,1000),
+(@GUID,36,1079.501,-678.1235,-160.946,100,2000),
+(@GUID,37,1080.827,-681.6636,-159.8325,100,6000),
+(@GUID,38,1085.063,-680.338,-158.8806,100,8000),
+(@GUID,39,1079.092,-677.261,-161.242,100,1000),
+(@GUID,40,1079.541,-677.6991,-161.0217,100,1000),
+(@GUID,41,1081.284,-682.4361,-159.5412,100,2000),
+(@GUID,42,1080.757,-681.0048,-159.9885,100,1000),
+(@GUID,43,1083.066,-680.2807,-159.4699,100,6000),
+(@GUID,44,1081.552,-677.1035,-160.5629,100,1000),
+(@GUID,45,1081.611,-680.4688,-159.852,100,11000),
+(@GUID,46,1080.048,-679.4224,-160.5198,100,7000),
+(@GUID,47,1082.761,-681.9189,-159.2203,100,11000),
+(@GUID,48,1078.952,-678.8635,-160.9521,100,8000),
+(@GUID,49,1081.18,-681.5133,-159.7614,100,1000),
+(@GUID,50,1080.676,-681.3997,-159.9307,100,11000),
+(@GUID,51,1078.617,-679.6107,-160.8951,100,1000),
+(@GUID,52,1081.279,-680.4371,-159.9548,100,1000),
+(@GUID,53,1078.764,-677.3422,-161.3203,100,1000),
+(@GUID,54,1080.66,-681.4279,-159.9294,100,2000),
+(@GUID,55,1080.101,-708.452,-154.2289,100,1000),
+(@GUID,56,1084.528,-731.8951,-151.01180,0,0),
+(@GUID,57,1093.696,-748.0836,-149.26190,0,0),
+(@GUID,58,1110.641,-738.1426,-146.50050,0,0),
+(@GUID,59,1115.694,-716.585,-144.63470,0,0),
+(@GUID,60,1148.22,-713.5815,-132.9880,0,0),
+(@GUID,61,1161.699,-694.4932,-132.20030,0,0),
+(@GUID,62,1149.113,-669.7729,-130.33960,0,0),
+(@GUID,63,1142.684,-647.411,-131.92330,0,0),
+(@GUID,64,1142.684,-647.411,-131.9233,1.53589,100000);
+
+-- ************************
+-- Baron Geddon
+-- ************************
+
+-- Pathing for Baron Geddon Entry: 12056 'TDB FORMAT' 
+-- He had incorrect points and he wasn't running
+DELETE FROM `dbscripts_on_creature_movement` WHERE `id` IN(5665501);
+INSERT INTO `dbscripts_on_creature_movement` (`id`, `delay`, `command`, `datalong`, `datalong2`, `buddy_entry`, `search_radius`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `comments`) VALUES 
+(5665501,0,25,1,0,0,0,0,0,0,0,0,0,0,0,0,'Baron Geddon - Run');
+
+-- Change his movement speed to the values from tbc-db
+-- Judging from my observations from retail he is always in runmode (you can see his animation is slightly different), and classic-db's SpeedRun=3,15 was too fast
+UPDATE `creature_template` SET `SpeedWalk`=1, `SpeedRun`=2.28571;

```

Comment:
I assume you're asking where he had incorrect points?

He didn't walk all the way back to his rune like he should be.